x-pack/filebeat/input/streaming/input_manager.go

```diff
@@ -35,7 +35,10 @@ func NewInputManager(log *logp.Logger, store statestore.States) InputManager {
 }
 
 func cursorConfigure(cfg *conf.C) ([]inputcursor.Source, inputcursor.Input, error) {
-	src := &source{cfg: defaultConfig()}
+	dc := defaultConfig()
+	// set readControlDeadline to 3x the writeControlDeadline
```

Comment:
this is a standard convention

Reply:
I should reframe the statement here for better clarity, it is generally the case that `readControlDeadline > writeControlDeadline` since we need to give the ws server some time to respond with a `pong control frame`. The `3x` amount was my choice since it seems like a nice buffer accounting for network jitter.